doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -218,7 +218,9 @@ Keys samples
 Matter samples
 --------------
 
-|no_changes_yet_note|
+   * Added Test Event Triggers to all Matter samples.
+     By utilizing the Test Event Triggers, you can simulate various operational conditions and responses in your Matter device without the need for external setup.
+     To get started with using Test Event Triggers in your Matter samples and to understand the full capabilities of this feature, please refer to the guide available in the :ref:`matter_test_event_triggers` page.
```

Comment:
```suggestion
* Added test event triggers to all Matter samples.
  By utilizing the test event triggers, you can simulate various operational conditions and responses in your Matter device without the need for external setup.

  To get started with using test event triggers in your Matter samples and to understand the capabilities of this feature, refer to the :ref:`matter_test_event_triggers` page.
```
The change to indentation hides it from the diff, but there's some wording changes as well.